src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -310,6 +310,21 @@ struct EvolutionMetavars {
                      volume_dim, ::Frame::Inertial>,
                  GeneralizedHarmonic::Tags::ThreeIndexConstraintCompute<
                      volume_dim, ::Frame::Inertial>,
+                 GeneralizedHarmonic::Tags::DerivSpatialMetricCompute<
+                     volume_dim, ::Frame::Inertial>,
```

Comment:
Future improvement: shouldn't you be able to get the first derivative analytically from existing variables, like spacetime_metric and kappa?

Reply:
No kappa in spectre, but spacetime_metric+pi, yes :)